source/declarations.tex

```diff
@@ -2935,8 +2935,12 @@
   using C::x;
   int x(double);
 };
-int f(D* d) {
-  return d->x();    // ambiguous: \tcode{B::x} or \tcode{C::x}
+void f(D* d) {
+   d->x();       // ambiguous: \tcode{x}'s direct parent \tcode{A} is an ambiguous base of the naming class \tcode{D}
```

Comment:
Thanks for the feedback.  I made some changes.  Let me know if you folks think this is better or worse or too much?  Also do you still need a diffpdf (if so, if you could point me to any resources that show a non-cumbersome way to do it in windows would certainly be appreciated)?